src/Features/CSharp/Portable/CodeFixes/AddExplicitCast/InheritanceDistanceComparer.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CodeAnalysis.CSharp.CodeFixes.AddExplicitCast
+{
+    internal sealed class InheritanceDistanceComparer : IComparer<ITypeSymbol>
```

Comment:
Consider doc'ing this type.  i.e. mention a couple of examples, and explain why the user woudl prefer we pick one type over the other.